lib/src/tasks/gen_test_runner/api.dart

```diff
@@ -0,0 +1,139 @@
+// Copyright 2015 Workiva Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+library dart_dev.src.tasks.gen_test_runner.api;
+
+import 'dart:async';
+import 'dart:io';
+
+import 'package:dart_dev/src/tasks/gen_test_runner/config.dart';
+import 'package:dart_dev/src/tasks/task.dart';
+
+Future<GenTestRunnerTask> genTestRunner(
+    {String directory,
+    Environment environment,
+    String filename,
+    bool genHtml,
+    bool react,
+    List<String> scriptTags}) async {
+  var executable = 'gen-test-runner';
```

Comment:
I guess I'm confused then, didn't you say here that this is no longer a standalone executable: https://github.com/Workiva/dart_dev/pull/124/files#r51058202

The `GenTestRunnerTask` below doesn't do anything with the generated executable command - just stores it and outputs it later I assume. I'm fine with keeping the args so that they can be logged later, but building an executable command out of them that doesn't actually work is misleading